packages/rspack/src/builtin-plugin/css-extract/hmr/normalize-url.ts

```diff
@@ -1,32 +1,4 @@
-/* eslint-disable */
-
-/**
- * @param {string[]} pathComponents
- * @returns {string}
- */
-function normalizeUrl(pathComponents) {
-	return pathComponents
-		.reduce(function (accumulator, item) {
-			switch (item) {
-				case "..":
-					accumulator.pop();
-					break;
-				case ".":
-					break;
-				default:
-					accumulator.push(item);
-			}
-
-			return accumulator;
-		}, /** @type {string[]} */ ([]))
-		.join("/");
-}
-
-/**
- * @param {string} urlString
- * @returns {string}
- */
-module.exports = function (urlString) {
+module.exports = function (urlString: string): string {
```

Comment:
we have already inline normalizeUrl fn in `hotModuleReplacement.ts`, so can this file be deleted?